otwcode/otwarchive
# AO3-6381 Restrict admin access to spam page

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6381

## Purpose

What does this PR do?
Restrict admin access to spam

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?
Description 
Log in as an admin with the policy_and_abuse or superadmin role

You should see Spam in the admin navigation and it should take you to https://test.archiveofourown.org/admin/spam

You should be able to perform the various marking/unmarking as spam activities on this page as well as the Confirmed Spam and Approved Works pages

 

Log in as an admin without the policy_and_abuse or superadmin role

You should not see Spam in the admin navigation and should not be able to access https://test.archiveofourown.org/admin/spam 

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?
Cesium-Ice,  she/her
If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.

*Please note that if you do not fill in this section, we will use your GitHub account name and
they/them pronouns.*
